### ISSUE TITLE
MD013 false positive with ul, checkbox, link, and italicized trailing text

### ISSUE THREAD

**Issue by hepcat72** [DavidAnson/markdownlint#379]
It seems to me that the following should not generate a `MD013` error:

Generates an error:
```
- [ ] [Migrations created & committed](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process) *(or no model changes)*
```

Since this does not generate an error:

Does not generate an error:
```
- [ ] [Migrations created & committed *(or no model changes)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)
- [ ] [Tests implemented](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
```

Error:
```
.github/pull_request_template.md:29:81 MD013/line-length Line length [Expected: 80; Actual: 167]
```

**Comment by DavidAnson** [comment]
In its default configuration, MD013 does not report a violation if there are no spaces after the threshold. As explained in the documentation, this is to allow for things like long URLs (your second scenario). In the case of your first scenario, the extra text after the URL includes spaces and could be wrapped. There are other modes for this rule if you wish to be more or less strict: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md013

**Comment by hepcat72** [comment]
I see.  Is there a rule to not include URL-related characters that are not rendered, in the count, such as `[`, `]`, and `(*)`?

**Comment by DavidAnson** [comment]
No, this rule is about how the Markdown file looks in the editor. Some people prefer not to have very long lines and this rule can help enforce that. If you are not worried about that part of the editing experience or use word wrap in your editor, it can be safely disabled without affecting the end-user experience.
